ports/cpprealm/portfile.cmake

```
@@ -0,0 +1,44 @@
+vcpkg_from_github(
+        OUT_SOURCE_PATH SOURCE_PATH
+        REPO realm/realm-cpp
+        REF "v${VERSION}"
+        SHA512 "d880daea7bceb3143052cae9d8883b448162484adacc566cca5315ac096964d0c79c421b25a76d5c2efbd8c3294dfb5acd3dd7314a6c154163e7d141b9eb2e51"
+        HEAD_REF "main")
+
+set(CPPREALM_CMAKE_OPTIONS -DREALM_CPP_NO_TESTS=ON -DREALM_ENABLE_EXPERIMENTAL=ON -DREALMCXX_VERSION="2.1.0")
+
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"
+    DISABLE_PARALLEL_CONFIGURE
+    OPTIONS
+        ${CPPREALM_CMAKE_OPTIONS}
+)

```

Keep it simple. Quote for CMake, not for the command line interpreter.
```suggestion
vcpkg_cmake_configure(
    SOURCE_PATH "${SOURCE_PATH}"
    DISABLE_PARALLEL_CONFIGURE
    OPTIONS
        -DREALM_CPP_NO_TESTS=ON
        -DREALM_ENABLE_EXPERIMENTAL=ON
        -DREALMCXX_VERSION=${VERSION}
)
```